src/components/networked-hand-controls.js

```diff
@@ -153,38 +106,269 @@ AFRAME.registerComponent('networked-hand-controls', {
+        ) {
+      console.warn("fallback path: will add and then disable pose tracking", this.data.controllerComponent, this.el, this.el.components);
+
+      // however, we do still want _one_ event listener, so we add the button meshes
+      this.el.addEventListener('model-loaded', this.el.components[this.data.controllerComponent].onModelLoaded);
+
+      // here we have no loadModel() to use directly, so we fully inject (bypassing controller detection)
+      this.el.components[this.data.controllerComponent].injectTrackedControls({profiles:JSON.parse(this.data.webxrControllerProfiles)});
+      this.disableLoadedPoseTracking();
+    } 
+    else {
+      // ideal, default path, for controller components (like oculus-touch-controls), that have separate loadModel() function
+
+      // this prevents injectTrackedControllers from running, which prevents the 
+      // tracked-controls-webxr component from being added, which is responsible for pose tracking
+      // (which we are disabling, since we want this controller to follow this NAF entity instead)
+      this.el.components[this.data.controllerComponent].checkIfControllerPresent = x => x;
+      this.el.components[this.data.controllerComponent].injectTrackedControls = x => x;
+      
+      // we load the model indicated by the remote user's headset
+      this.el.components[this.data.controllerComponent].loadModel({profiles:JSON.parse(this.data.webxrControllerProfiles)});      
+      this.el.play();
+    }
+
+    this.injectedController = true;
+  },
+
+  disableLoadedPoseTracking() {
+    // handling for older controller components (like vive-controls) that lack the cleaner hooks 
+    // to block local post tracking. We allow pose tracking to start (so we can get loaded model), 
+    // and then stop it. Note: we do not attempt to handle older tracked-controls-webvr.
+    let counter = 0;
+    let disableInterval = setInterval(() => {
+      counter++;
+      if (this.el.components['tracked-controls-webxr']) {
+        // there is no remove on this function in current iteration, so we kill it with pause and tick first.
+        this.el.components['tracked-controls-webxr'].pause(); 
+        this.el.components['tracked-controls-webxr'].tick = () => {};
+        this.el.removeAttribute('tracked-controls-webxr'); // just some cleanup for inspector, etc.
+
+        clearInterval(disableInterval);
+        this.el.play();
+      }
+      if (counter > 1000) {
+        // 25 seconds, don't want the possiblity of this running forever in the background
+        clearInterval(disableInterval);
+        try {
+          this.el.components[this.data.controllerComponent].remove();
+        } catch(e) {
+          console.error("error removing controller component", e);
+        }
+        throw new Error("never found tracked-controls-webxr, cancelling interval that was waiting to remove it and removing controller components")
+      }
+    },25)
```

Comment:
Can you please use the tick method instead of setInterval?

Reply:
I _think_ it's better to not do that in this case. `this.tick()` already has a job. This `setInterval()` will usually run once, but it absolutely will only run for maximum 25~ seconds if there's an error. It will always be removed. To add it to the tick, I'd need to set a flag in init, add a check on that flag on every tick at all times (even when this code isn't relevant--which, since there aren't many people using these old controllers, will usually be the case), and turn that flag on/off here.

The "optimization" of using tick() here seems to me like it would actually hurt, not help. Agreed?

Reply:
to be clear, ideally 
1. we wouldn't have this at all (which, we usually don't, if the component separates pose tracking and model injection and allows us bypass one without the other)
2. fallback: would be nice if we could 'listen' for when the tracking component gets added and remove it on-that-event, but there is no such event
3. we could do a 'setTimeout', but that would be gross; so, we do a 'setInterval' instead, on a short loop; while this setInterval almost always runs exactly one time in my testing, a setInterval provides a layer of safety in case something goes wrong.

Reply:
I did make some updates to this code. In testing, I was able to see that it usually only looped once even with a 1ms interval. I went ahead and switched it out to a `requestAnimationFrame` rather than do a 1ms interval, just because a 1ms interval if it does run seems like it could produce too much overhead if it were to run its full length, even if it only would be for a few seconds.